packages/experimental/MenuButton/src/MenuButton.styling.ts

```diff
@@ -0,0 +1,26 @@
+import { UseStylingOptions, buildProps, Theme } from '@fluentui-react-native/framework';
+import { borderStyles, layoutStyles, shadowStyles, fontStyles } from '@fluentui-react-native/tokens';
+import { menuButtonName, MenuButtonProps, MenuButtonSlotProps, MenuButtonTokens } from './MenuButton.types';
+import { defaultButtonTokens, buttonStates } from '@fluentui-react-native/experimental-button/src/ButtonTokens';
+
+export const stylingSettings: UseStylingOptions<MenuButtonProps, MenuButtonSlotProps, MenuButtonTokens> = {
+  tokens: [menuButtonName, defaultButtonTokens],
+  states: buttonStates,
+  slotProps: {
+    button: buildProps(
+      (tokens: MenuButtonTokens, theme: Theme, props: MenuButtonProps) => ({
+        style: {
+          backgroundColor: tokens.backgroundColor,
+          color: tokens.color,
+          iconColor: tokens.iconColor,
+          ...borderStyles.from(tokens, theme),
```

Comment:
Anything passed into here will be fed into button as its style prop, which gets passed into its root slot. This is why applying font and text related styling isn't working.

We may need to change button to expose some of its tokens as props, such as the foreground color/font tokens.

Reply:
Another option that could avoid some of this is to use Button.customize inside MenuButton, but I haven't fully thought that option out so that might run into some pitfalls.

Reply:
I'll try to make it first way

Reply:
Fixed